### PR TITLE
feat(ipfs-topology): Remove gateway multiaddrs from bootstrap list

### PR DESCRIPTION
**Pull request by stbrody**
I'm not totally sure who's owning the gateway deprecation.  Put @oed and @federicorinaldi as the approvers for now.

CC @smrz2001 @v-stickykeys @willex

### CODE REVIEW

File: packages/core/src/__tests__/ceramic.test.ts [hunk 1]
```diff
@@ -245,7 +245,7 @@ describe('Ceramic integration', () => {
           /rejected because it builds on stale state/
         )
         expect(streamOg.state.log.length).toEqual(2)
-
+        f
```
[the remark this answers is not in the document]
Comment (stbrody) [reply packages/core/src/__tests__/ceramic.test.ts:248]: wut.  How did that get in there?!  😅

Comment (stbrody) [inline packages/core/src/__tests__/ceramic.test.ts:248]:
```suggestion

```

File: packages/core/src/__tests__/ceramic.test.ts [hunk 2]
```diff
@@ -245,7 +245,6 @@ describe('Ceramic integration', () => {
           /rejected because it builds on stale state/
         )
         expect(streamOg.state.log.length).toEqual(2)
-
         // While we disallow creating commits based on a stale tip as part of a user request when the node already
```
Comment (stbrody) [inline packages/core/src/__tests__/ceramic.test.ts:248]:
```suggestion

        // While we disallow creating commits based on a stale tip as part of a user request when the node already
```

**Comment by oed** [comment]
@federicorinaldi owns it. Not sure if this was accounted for. lftm

**Review by federicorinaldi — APPROVED** [review]

**Pull request merged**